apps/transmitmedia.cpp

```diff
@@ -684,21 +690,21 @@ class ConsoleSource: public Source
 #endif
     }
 
-    int Read(size_t chunk, bytevector& data, ostream & ignored SRT_ATR_UNUSED = cout) override
+    int Read(size_t chunk, MediaPacket& pkt, ostream & ignored SRT_ATR_UNUSED = cout) override
     {
-        if (data.size() < chunk)
-            data.resize(chunk);
+        if (pkt.payload.size() < chunk)
+            pkt.payload.resize(chunk);
 
-        bool st = cin.read(data.data(), chunk).good();
+        bool st = cin.read(pkt.payload.data(), chunk).good();
```

Comment:
For `ConsoleSource` class the time should be also grabbed. This is because the use cases embrace also passing payload through standard input and this is also considered live playing.